api/opencl_runtime_layer.txt

```diff
@@ -938,7 +870,7 @@ Otherwise, it returns one of the following errors:
 [NOTE]
 ====
 Calling *clEnqueueReadBuffer* to read a region of the buffer object with the
-_ptr_ argument value set to _host_ptr_ + _offset_, where _host_ptr_ is a
+_ptr_ argument value set to _host_ptr_  _offset_, where _host_ptr_ is a
```

Comment:
Regarding ` _host_ptr_ + _offset_` to ` _host_ptr_ _offset_`: I think the plus sign is important here and should not be removed.  If it is causing formatting issues, maybe spell it out?  Something like: ` _host_ptr_ plus _offset_`?

Reply:
Fixed. Not sure how that happened.